sdk/maps/azure-maps-render/README.md

```diff
@@ -0,0 +1,165 @@
+```
+
+Get Map Static Image
+```java com.azure.maps.render.sync.get_map_static_image
+System.out.println("Get Map Static Image");
+GeoBoundingBox bbox = new GeoBoundingBox(1.355233, 42.982261, 24.980233, 56.526017);
+new StaticMapLayer();
+new RasterTileFormat();
+MapStaticImageOptions mapStaticImageOptions = new MapStaticImageOptions().setStaticMapLayer(StaticMapLayer.BASIC)
+    .setMapImageStyle(MapImageStyle.MAIN).setZoom(2)
+    .setBoundingBox(bbox).setRasterTileFormat(RasterTileFormat.PNG);
+client.getMapStaticImage(mapStaticImageOptions).toStream();
+```
+
+Get Copyright From Bounding Box
+```java com.azure.maps.render.sync.get_copyright_from_bounding_box
+GeoBoundingBox boundingBox = new GeoBoundingBox(52.41064, 4.84228, 52.41072, 4.84239);
+client.getCopyrightFromBoundingBox(boundingBox, true);
+```
+
+Get Copyright For Tile
+```java com.azure.maps.render.sync.get_copyright_for_tile
+client.getCopyrightForTile(new TileIndex().setX(9).setY(22).setZ(6), true);
+```
+
+Get Copyright For World
+```java com.azure.maps.render.sync.get_copyright_for_world
+client.getCopyrightForWorld(true);
+```
+
+## Troubleshooting
+When you interact with the Azure Maps Services, errors returned by the Language service correspond to the same HTTP status codes returned for REST API requests.
+
+For example, if you search with an invalid coordinate, a error is returned, indicating "Bad Request".400
+
+## Next steps
+
+Several Azure Maps Render Java SDK samples are available to you in the SDK's GitHub repository.
+[Azure Maps Render Samples](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/maps/azure-maps-render/src/samples)
+
+## Contributing
+
+For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md).
+
+1. Fork it
+1. Create your feature branch (`git checkout -b my-new-feature`)
+1. Commit your changes (`git commit -am 'Add some feature'`)
+1. Push to the branch (`git push origin my-new-feature`)
+1. Create new Pull Request
+
+<!-- LINKS -->
+[source]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/maps/azure-maps-render/src
+[samples]:  https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/maps/azure-maps-render/src/samples
+[rest_docs]: https://docs.microsoft.com/rest/api/maps
+[product_docs]: https://docs.microsoft.com/azure/azure-maps/
+[docs]: https://azure.github.io/azure-sdk-for-java/
+[jdk]: https://docs.microsoft.com/java/azure/jdk/
+[azure_subscription]: https://azure.microsoft.com/free/
+[azure_identity]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/identity/azure-identity
+[azure_core_http_netty]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-http-netty
+[authenticate]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/docs/AUTH.md
+[design]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/docs/DESIGN.md
```

Comment:
Missing `impressions` tag at the end of the file. See the README of azure-data-tables for example.